tutorial.md

```diff
@@ -148,9 +148,7 @@ Be sure to add the new resources to your `flutter.yaml` file.
 
```

Comment:
Hmm. Actually `flutter create` is not mentionned in this tutorial. So I think a section about `flutter.yaml` should remain until `flutter create` appears here.